### ISSUE TITLE
Add mailing list to Gmane

### ISSUE THREAD

**Issue by pank** [bokeh/bokeh#2371]
Hi,

It would be great it the mailing list would be mirrored on Gmane as the google interface is not so great, and probably can't be subscribed via NNTP (I don't know how at least).

It's best if an administrator who makes the request, I guess.  

The link to register is: http://gmane.org/subscribe.php
Mailing list software should be set to google.  The gmane address should be something like 
`gmane.comp.python.bokeh`, I guess.

Thanks,
Rasmus

**Comment by bryevdv** [comment]
What sort of posting permissions are you proposing?

**Comment by pank** [comment]
Bryan Van de Ven <notifications@github.com> writes:

> What sort of posting permissions are you proposing?

Full posting, I guess.

Plenty of Python projects MLs are mirrored on gmane:

    http://dir.gmane.org/index.php?prefix=gmane.comp.python

**Comment by bryevdv** [comment]
@pank the mailing list itself requires new users to be approved (because we get spam) so how does that work if new users post to gmane first? I guess that just gets forwarded to google groups, and the mods get the same notification to approvem and after approval the post get mirrored back to gmane?

**Comment by bryevdv** [comment]
@damianavila @bokeh/core anyone have any thoughts on this?

**Comment by damianavila** [comment]
I don't know how gname works... but I heard that the ability to search into post is nice... I know that other projects just use it as a kind of backup and because it is easy to navigate a look for things, but not too much more info. I see problem if this is not in sync with the google group, I don't want to open another channel and then forgot people asking things there... if there is some sync, then I think is not harm to add it.

**Comment by pank** [comment]
Damián Avila <notifications@github.com> writes:

> I don't know how gname works...

It's simply a mailing list to NNTP﻿¹ proxy.  So that you can read your
mailing lists via your news client.  Further, it works as a "backup" or
mirror of a mailing list.  And potentially as a posting gateway.

> but I heard that the ability to search into post is nice...

I'm not sure that's a good reason.  I mean, google should be awesome at
searching...

> I see problem if this is not in sync with the google group, I don't want
> to open another channel and then forgot people asking things there... if
> there is some sync, then I think is not harm to add it.

There is sync and usually it's fast.  At times there's delayed in the
ML→News process.  A message is never posted on Gmane "before" it's posted
on the mailing list so here the delay would be short, I imagine.

Rasmus

Footnotes: 
﻿¹   https://en.wikipedia.org/wiki/Network_News_Transfer_Protocol

-- 
Lasciate ogni speranza, voi che leggete questo.

**Comment by pank** [comment]
Bryan Van de Ven <notifications@github.com> writes:

> @pank the mailing list itself requires new users to be approved
> (because we get spam) so how does that work if new users post to gmane
> first?

One ML list I used to check is members only.  I got the following message
when posting to it:

> From: Gmane Autoauthorizer 
> Subject: ML: Authorization required
>
> This is a non-public mailing list, which means that you have to
> subscribe to the list to post to it.  If you're already subscribed to
> the list, Gmane can forward the message you sent to the list if you respond 
> to this message.  If not, you should sign up to the mailing list first,
> and then respond to this message, or just forget about it.
>
> Many mailing lists have an option to subscribe to a list, but then put
> it in 'nomail' mode, which means that you won't receive any mail from
> the list.  
> 
> [INSTRUCTIONS HOW TO SUBSCRIBE GIVEN UNDERLYING SOFTWARE]

I can only assume it works with underlying google software as well.

> I guess that just gets forwarded to google groups, and the mods
> get the same notification to approvem and after approval the post get
> mirrored back to gmane?

I don't know.  I don't think a post would ever show up on Gmane before
showing up on the mailing list itself.

On spam: according to Wikipedia, Gmane does spam filtration on both
incoming and outgoing messages.  The FSF hosted mailing lists have
basically zero spam (maybe one every few months).

Rasmus

-- 
Together we will make the possible totalllly impossible!

**Comment by bryevdv** [comment]
I have submitted the application.
